### DISCUSSION TITLE
OpenTracing support

### DISCUSSION THREAD

**Discussion by alkistis500** [jaegertracing/jaeger#4250] [Q&A] [answered]
Hello,

I have noticed in Client Libraries documentation that they have been deprecated.
https://www.jaegertracing.io/docs/1.42/client-libraries/
This is the case for older versions as well.

For existing applications that are already instrumented with the OpenTracing API does latest Jaeger version support receiving OpenTracing traces?

Thank you in advance

**Comment by yurishkuro** [comment, marked as answer]
There is no such thing as OpenTracing traces, since OpenTracing was just an API, it never specified a data format. The data format that Jaeger accepts was dictated by the Jaeger clients (SDKs), as documented in the jaeger-idl repository. That format is still supported and will remain supported for a long time. However, aside from the deprecated Jaeger SDKs, nothing really will be producing that format, because even though OpenTelemetry SDKs are capable of producing it now, that support is being phased in out OTEL since Jaeger can accept OTEL native OTLP format directly.

**Reply by alkistis500** [reply]
Thanks for the prompt and detailed answer. 
So, Jaeger Client are still supported even if they are deprecated. Correct?

**Reply by yurishkuro** [reply]
The *format* is supported. You can continue using Jaeger clients, they will still work, but there will be no new versions / upgrades.
